js/apps/kc-create/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "kc-create",
+  "version": "999.0.0-SNAPSHOT",
+  "description": "Create a new Keycloak ui project based on a template",
+  "main": "create.js",
+  "type": "module",
+  "scripts": {
+    "start": "node create.js"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@types/node": "^20.14.2"
+  },
+  "dependencies": {
+    "chalk": "^5.3.0",
+    "commander": "^12.1.0",
+    "fs-extra": "^11.2.0",
+    "mustache": "^4.2.0",
+    "simple-git": "^3.25.0"
+  },
```

This JSON is malformed.

```suggestion
  }
```